### PR TITLE
Two changes for implementing playground in Suneido

### PR DESCRIPTION
**Pull request by Hao-Axon**

### CODE REVIEW

File: runtime/builtin/UI/suNode.ts [hunk 1]
```diff
@@ -49,6 +62,17 @@ export class SuElement extends SuNode {
     type(): string {
         return 'Element';
     }
+    GetBoundingClientRect() {
+        let rect = this.el.getBoundingClientRect();
+        let map = new Map();
```
Comment (apmckinlay) [inline runtime/builtin/UI/suNode.ts]: Was there a reason you used a different way of constructing the map from the two above?

Comment (Hao-Axon) [reply runtime/builtin/UI/suNode.ts]: Not really. In AddEventListener method, I probably did that way just in order to make the SuObject creation in one statement. I can make them be consistent.

Comment (apmckinlay) [reply runtime/builtin/UI/suNode.ts]: Ok, no big deal, I was just curious. Would it make sense to have an SuObject constructor that took a hash so you could write it more naturally?

Comment (Hao-Axon) [reply runtime/builtin/UI/suNode.ts]: Not sure what the 'hash' you mentioned means. Do you mean a JS Object? A problem of the JS Object is that its key must be a string, which will limit the new constructor's capability.

We already have a test util function in testUtitlity.ts
```javascript
function makeObj(vec: any[], ...mapPairs: [any, any][]): SuObject {
    return new SuObject(vec, new Map(mapPairs));
}
```
Do you think it is ok to make this a static method in SuObject?

Comment (apmckinlay) [reply runtime/builtin/UI/suNode.ts]: Yes, I was thinking of a JS object {...}, but I forgot about the string key limitation. We could use something like makeObj but it doesn't save much.  A static method would work. Not a big deal, it was just a thought. Whatever you think is fine.

Comment (Hao-Axon) [reply runtime/builtin/UI/suNode.ts]: Yes, it won't help too much. 
I think I will just make the two calls consistent for now.

**Pull request merged**